internal/cmd/base/servers.go

```diff
@@ -342,6 +345,29 @@ func (b *Server) RemovePidFile(pidPath string) error {
 	return os.Remove(pidPath)
 }
 
+func (b *Server) StoreWorkerAuthReq(authReq, workerAuthReqPath string) error {
+	// Quit fast if no workerAuthReqFile
+	if workerAuthReqPath == "" {
+		return nil
+	}
+	workerAuthReqFilePath := fmt.Sprintf("%s/%s", workerAuthReqPath, WorkerAuthReqFile)
+
+	// Open the workerAuthReq file
+	workerAuthReqFile, err := os.OpenFile(workerAuthReqFilePath, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, 0o644)
```

Comment:
I think `0o640` is probably fine. The request isn't secret, but some might consider it sensitive.

Reply:
This got resolved but as there was no comment but also no change it feels like maybe it was resolved by accident. If you want to leave it that's fine, can you just confirm that you want to leave it as-is?

Reply:
D'oh my bad, I thought the existing code had the same number you commented. Fixed!